tests/test_dispatcher/test_deprecated.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from aiogram.dispatcher.event.observer import TelegramEventObserver
+from aiogram.dispatcher.router import Router
+
+OBSERVERS = {
+    "callback_query",
+    "channel_post",
+    "chosen_inline_result",
+    "edited_channel_post",
+    "edited_message",
+    "errors",
+    "inline_query",
+    "message",
+    "poll",
+    "poll_answer",
+    "pre_checkout_query",
+    "shipping_query",
+    "update",
+}
+
+
+def test_deprecated_handlers_name():
+    from aiogram import __version__
+
+    major, minor = map(int, __version__.split(".")[:-1])
+
+    if minor >= 2 and major >= 3:  # version >=3.2.*
```

Comment:
I think this validation can be moved to separated function because it can be used in other places

Make something like this:
```python
@contextmanager
def check_deprecated(
    max_version: str,
    warning: Type[Warning] = DeprecationWarning,
    exception: Type[Exception] = AttributeError,
) -> None:
    parsed_max_version = version.parse(max_version)
    current_version = version.parse(aiogram.__version__)

    if parsed_max_version < current_version:
        with pytest.raises(exception):
            yield
    else:
        with pytest.warns(warning, match=max_version):
            yield
```

In this case `packaging` should be added as development dependency (`poetry add -D packaging`).

And then it can be used like this:
```python
with check_deprecated('3.2'):
    getattr(<obj>, <attribute>)
```